.github/workflows/ci.yml

```diff
@@ -0,0 +1,14 @@
+on:
+  push:
+
+name: CI
+
+jobs:
+
+  jsonlint:
+    name: Build and test
+    runs-on: ubuntu-latest
+    steps:
+      - run: sudo apt-get install -y python3-demjson
+      - uses: actions/checkout@v1
```

Comment:
There's now a `v2` for the checkout action.

Reply:
Fixed.